neuro_extras/main.py

```diff
@@ -78,7 +84,7 @@ def data_transfer(source: str, destination: str) -> None:
     """
     Copy data between storages on different clusters.
     """
-    run_async(_transfer_data(source, destination))
+    run_async(_data_transfer(source, destination))
```

Comment:
side change: rename for consistency: everything in code should be called "data transfer" not "transfer data" so that we can find it easily.